### PR TITLE
Attaching a new container to an IV drip automatically swap containers

### PR DESCRIPTION
**Pull request by ShiftyRail**
This PR was revealed to me in a dream.
[qol]

:cl:
- tweak: Attaching a new container to an IV drip or cryo cell automatically swaps containers if the previous one was empty.

### CODE REVIEW

**Comment by gurfan** [comment]
Best PR of 23

**Comment by Securitymain** [comment]
Does remove_container() swap it with the item in the users hand or does it just drop?

Personally I would have made it always swap the two
Cryo also needs this

**Comment by ShiftyRail** [comment]
It drops it. I can make it swap.

**Comment by xXxEMBERGHOSTSQUADDIExXx** [comment]
why not have it always swap even if its not empty?

**Comment by Dacendeth** [comment]
The empty requirement feels silly, if there's 5u of chems remaining having to manually detach just feels needlessly clunky.

**Comment by ShiftyRail** [comment]
Done for autoswap in hand & for cryo.

File: code/game/machinery/cryo.dm [hunk 1]
```diff
@@ -317,11 +317,10 @@ var/global/list/cryo_health_indicator = list(	"full" = image("icon" = 'icons/obj
 		detach()
 	return ..()
 
-/obj/machinery/atmospherics/unary/cryo_cell/attackby(var/obj/item/weapon/G as obj, var/mob/user as mob)
+/obj/machinery/atmospherics/unary/cryo_cell/attackby(var/obj/item/weapon/G, var/mob/user)
```
Comment (Kurfursten) [inline code/game/machinery/cryo.dm:320]: substantive change not in cl

**Comment by Kurfursten** [comment]
fixed it for you

**Pull request merged**